### DISCUSSION TITLE
`tweaked.cc` category separation

### DISCUSSION THREAD

**Discussion by SkyTheCodeMaster** [cc-tweaked/CC-Tweaked#753] [Q&A] [answered]
How does [tweaked.cc](https://tweaked.cc) separate it's categories into `Globals`, `Modules`, `Peripherals`, etc?
In the [illuaminate.sexp](https://github.com/SquidDev-CC/CC-Tweaked/blob/mc-1.15.x/illuaminate.sexp#L27-L30) I see that it's defined, and I've done something similar in my `illuaminate.sexp` file (not on github yet, sorry!), but I don't know how to separate the files.
In the CC-Tweaked files, it's just the normal [@module gps](https://github.com/SquidDev-CC/CC-Tweaked/blob/mc-1.15.x/src/main/resources/data/computercraft/lua/rom/apis/gps.lua#L23) or [@module cc.expect](https://github.com/SquidDev-CC/CC-Tweaked/blob/mc-1.15.x/src/main/resources/data/computercraft/lua/rom/modules/main/cc/expect.lua#L4), but they're separated on the website. How would I do this with illuaminate?

**Comment by SkyTheCodeMaster** [comment]
When writing a markdown file for illuaminate's doc-gen function, if there is no newline at the end, it errors with
`illuaminate: internal error, uncaught exception:
             Invalid_argument("offset 95 invalid: 0 <= offset < 38")`
Adding a newline fixes this error.

**Reply by SquidDev** [reply]
Would you be able to rerun your program with the `OCAMLRUNPARAM=b` environment variable set and report a bug on [illuaminate's repository](https://github.com/SquidDev/illuaminate)?

**Reply by SkyTheCodeMaster** [reply]
Done!
https://github.com/SquidDev/illuaminate/issues/23

**Comment by SkyTheCodeMaster** [comment, marked as answer]
The answer for this is that doing `@module[kind=category] name` works
